rosdep/python.yaml

```
@@ -5648,6 +5648,13 @@ python3-pytest:
   openembedded: [python3-pytest@meta-python]
   rhel: ['python%{python3_pkgversion}-pytest']
   ubuntu: [python3-pytest]
+python3-pytest-cov:
+  alpine: [py3-pytest-cov]
+  arch: [python-pytest-cov]
+  debian: [python3-pytest-cov]
+  fedora: [python3-pytest-cov]
+  gentoo: [dev-python/pytest-cov]
+  ubuntu: [python3-pytest-cov]
```

Also available in EPEL 7 and 8: https://apps.fedoraproject.org/packages/python-pytest-cov

```suggestion
  rhel: ['python%{python3_pkgversion}-pytest-cov']
  ubuntu: [python3-pytest-cov]
```

---

awesome thank you!